src/component/Dropdown.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useMemo } from 'react';
+import { Person } from '../types/Person';
+
+import { peopleFromServer } from '../data/people';
+import debounce from 'lodash.debounce';
+import classNames from 'classnames';
+
+type Props = {
+  onSelected: (person: string) => void;
+  delay: number;
+};
+
+export const Dropdown: React.FC<Props> = ({ onSelected, delay = 300 }) => {
+  const [query, setQuery] = useState('');
+  const [selected, setSelected] = useState(false);
+
+  const applyQuery = () => {
+    debounce(() => setSelected(true), delay)();
+  };
```

Comment:
- suggestions should appear after some delay in typing (debounce);
![image](https://github.com/mate-academy/react_autocomplete/assets/63243136/5ea15825-76e0-46b1-80e3-e3552dc6fb88)

You should set `appliedQuery` and filter people depending on it. Now it isn't working.
`selected` state is redundant and doesn't work.